features/256-auth-gear-interaction-overview-in-next/api.md

```diff
@@ -54,173 +54,6 @@ Following function will be removed from client SDK, a developer should create it
 
 For security reason, Cloud Function should invoke admin related API with master key.
 
-## Execution flow of hooked Cloud Function
```

Comment:
@kiootic just for readability sake, can you:

1. Add a section here called "Web Hook", with a sentence mention details of hook is in another spec (path)
2. Add a table similar to the one below, with all hooks supported.